installation.rst

```diff
@@ -3,3 +3,166 @@
+      especially if you are using Python for other purposes.
+    - In **OSX**, especially the latest versions that may have **System Integrity Protection**, pip may fail. Try using virtualenvs, or
+      install with another user ``$ pip install --user conan``.
+    - Some Linux distros, such as Linux Mint, require a restart (shell restart, or logout/system if not enough) after
+      installation, so Conan is found in the path.
+    - In Windows, Python 3 installation can fail installing the ``wrapt`` dependency because of a bug in **pip**. Information about this issue and
+      workarounds is available here: https://github.com/GrahamDumpleton/wrapt/issues/112.
+
+Known installation issues with pip
+++++++++++++++++++++++++++++++++++
+
+- When Conan is installed with :command:`pip install --user <username>`, usually a new directory is created for it. However, the directory
+  is not appended automatically to the `PATH` and the :command:`conan` commands do not work. This can usually be solved restarting the session of
+  the terminal or running the following command:
+
+  .. code-block:: bash
+
+      $ source ~/.profile
+
+Install from brew (OSX)
+-----------------------
+
+There is a brew recipe, so in OSX, you can install Conan as follows:
+
+.. code-block:: bash
+
+    $ brew update
+    $ brew install conan
+
+Install from AUR (Arch Linux)
+-----------------------------
+
+The easiest way to install Conan on Arch Linux is by using one of the `Arch User Repository (AUR) helpers <https://wiki.archlinux.org/index.php/AUR_helpers>`_, e.g., **yay**, **aurman**, or **pakku**.
+For example, the following command installs Conan using ``yay``:
+
+.. code-block:: bash
+
+    $ yay -S conan
+
+Alternatively, build and install Conan manually using ``makepkg`` and ``pacman`` as described in `the Arch Wiki <https://wiki.archlinux.org/index.php/Arch_User_Repository#Installing_and_upgrading_packages>`_.
+Conan build files can be downloaded from AUR: https://aur.archlinux.org/packages/conan/.
+Make sure to first install the three Conan dependencies which are also found in AUR:
+
+- python-patch-ng
+- python-node-semver
+- python-pluginbase
+
+
+Install the binaries
+--------------------
+
+Go to the conan website and `download the installer for your platform <https://conan.io/downloads.html>`_!
+
+Execute the installer. You don't need to install python.
+
+
+Initial configuration
+---------------------
+
+Check if Conan is installed correctly. Run the following command in your console:
+
+.. code-block:: bash
+
+    $ conan
+
+The response should be similar to:
+
+.. code-block:: bash
+
+    Consumer commands
+      install    Installs the requirements specified in a recipe (conanfile.py or conanfile.txt).
+      config     Manages Conan configuration.
+      get        Gets a file or list a directory of a given reference or package.
+      info       Gets information about the dependency graph of a recipe.
+      ...
+
+.. tip::
```

Comment:
Maybe remove this ref until we can check it and make sure it is updated to support Conan 2.0